analytical_engine/java/grape-runtime/pom.xml

```diff
@@ -64,14 +64,18 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
-    <dependency>
+      <dependency>
```

Comment:
Seems that the indention of newly added blocks is not aligned with existing dependencies..

Reply:
BTW I have noticed that we have switch forth and back the logger for many times. Is there any consideration for that?

Reply:
Actually in previous PR, the logging problem has not been fixed(Only `ERROR` log is visible), in this PR, we replace log4j.properties with log4j.xml and fix this issue.